### DISCUSSION TITLE
SCIM: Audit Logs for User related operation

### DISCUSSION THREAD

**Discussion by dhanan-np** [JanssenProject/jans#4790] [Q&A] [answered]
Hello Team,
Is it possible to configure audit logs related to User's operation via SCIM.
We basically want to have logs on all kind of operations performed and by whom it was performed.

**Comment by nynymike** [comment, marked as answer]
"by whom" is a strange concept in SCIM, because it is a REST API called by an OAuth client. If the OOTB logs don't have the information you want, you could add log information in the SCIM interception script. If there is a person assocaited with the Grant, you could back into a subject identity. See [Developer Guide](https://docs.jans.io/head/admin/developer/scripts/scim/) for info on the SCIM interception script.

**Reply by dhanan-np** [reply]
well by whom, I was intended to the "Token/IP-Address".
I will go through the doc and will comeback if I have any queries related to it

**Comment by ossdhaval** [comment]
Hi @dhanan-np if there are no further queries and if you have found your answer, can you mark the response as an answer and we can close this discussion.
